contribute/index.html

```diff
@@ -270,15 +270,15 @@ <h1 class="h3">
         </p>
         <h2 class="h5">Become a Backer</h2>
         <p>
-            Starting at only $1/month, you can join <a href="https://opencollective.com/hoodie">Hoodie’s OpenCollective</a>. Every contribution counts! And all expense are transparently documented.
+            Starting at only $1/month, you can join <a href="https://opencollective.com/hoodie">Hoodie’s OpenCollective</a>. Every contribution counts! All expense are transparently documented.
```

Comment:
can you please fix it yourself? I’m on a plan and they block git, and internet is very flaky